x-pack/plugins/apm/server/plugin.ts

```diff
@@ -235,6 +236,14 @@ export class APMPlugin
       logger: this.logger,
     });
 
+    // search strategies for async partial search results
+    core.getStartServices().then(() => {
```

Comment:
Why is this wrapped in `core.getStartServices`? There's no parameter (`coreStart`) and it looks like all methods in `setup`.

Reply:
I think this is just how it's supposed to be set up according to the [examples/documentation](https://github.com/elastic/kibana/blob/master/examples/search_examples/server/plugin.ts)

Reply:
In that example it needs something from the start deps, so I don't think we should be doing this if we don't need to.

Reply:
Yep you're right - Updated here https://github.com/elastic/kibana/commit/034c680090836471795efc4bf8e30f0db5806939